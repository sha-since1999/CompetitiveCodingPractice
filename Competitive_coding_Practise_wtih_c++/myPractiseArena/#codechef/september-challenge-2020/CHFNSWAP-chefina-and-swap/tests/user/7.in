3
1000000000
983475643
345769391