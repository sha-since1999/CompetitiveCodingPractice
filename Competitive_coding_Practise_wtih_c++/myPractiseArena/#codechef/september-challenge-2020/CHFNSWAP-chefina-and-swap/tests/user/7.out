292893219
288053347
101273511
